test/raven.client.js

```diff
@@ -1018,4 +1018,31 @@ describe('raven.middleware', function () {
     client2.should.not.equal(client1);
     client2.should.be.an.instanceof(raven.constructor);
   });
+
+  it('should explicitly add req and res to the domain', function (done) {
+    var client = new raven.Client(dsn).install();
+
+    var EventEmitter = require('events');
+    // node 0.10 compat
+    if (process.version <= 'v0.11') EventEmitter = EventEmitter.EventEmitter;
+    var e = new EventEmitter();
+    e.on('done', function () {
+      // Context won't propagate here without the explicit binding of req/res done in the middleware
+      setTimeout(function () {
+        client.getContext().breadcrumbs.length.should.equal(1);
```

Comment:
I assume that breadcrumbs[0].mesage==='test breadcrumb' here? is there a chance that some other one got added and this doesn't work?